iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/LineAdderAdapter.java

```diff
@@ -52,24 +57,84 @@
 
     private double b2 = Double.NaN;
 
+    private double p0 = Double.NaN;
+
+    private double q0 = Double.NaN;
+
+    private String ucteXnodeCode;
+
     LineAdderAdapter(final MergingViewIndex index) {
         this.index = Objects.requireNonNull(index, "merging view index is null");
     }
 
     @Override
     public Line add() {
-        Line newLine = null;
+        Line newLine;
         final Network n1 = checkAndGetNetwork1();
         final Network n2 = checkAndGetNetwork2();
         if (n1 == n2) {
             newLine = index.getLine(addLine(n1));
         } else {
+            // FIXME : how to compute ucteXnodeCode ?
+            ucteXnodeCode = "";
+            // P0 & Q0 are updated by MergingNetworkListener::onUpdate method
+            p0 = 0.0;
+            q0 = 0.0;
+
+            // Taking into account ensureIdUnicity
+            checkAndSetUniqueId();
             // Creation of 2 dangling lines
-            throw MergingView.NOT_IMPLEMENTED_EXCEPTION;
+            // -- first dangling line
+            final MergingView view = index.getView();
+            final VoltageLevel vl1 = view.getVoltageLevel(voltageLevelId1);
+            addDanglingLine(vl1, id + DL1_SUFFIX, connectableBus1, bus1, node1, g1, b1);
+            // -- second dangling line
+            final VoltageLevel vl2 = view.getVoltageLevel(voltageLevelId2);
+            addDanglingLine(vl2, id + DL2_SUFFIX, connectableBus2, bus2, node2, g2, b2);
+            // MergedLine.id is forced here
+            // Return the merged line as the new line
+            newLine = index.getMergedLineByCode(ucteXnodeCode)
+                           .setId(id);
         }
         return newLine;
     }
 
+    private void checkAndSetUniqueId() {
+        if (this.id == null) {
+            throw new PowsyblException(getClass().getSimpleName() + " id is not set");
```

Comment:
I'm not sure this is a good idea to use `getClass().getSimpleName()` to get "Line"